rfcs/text/0025-container-metric-fields.md

```diff
@@ -1,8 +1,8 @@
 # 0025: Container Metric Fields
 <!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
 
-- Stage: **1 (draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
-- Date: **2021-07-29** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+- Stage: **2 (draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **2022-02-02** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
```

Comment:
This should be the date when this PR gets merged. But no worries, it will get changed later 🙂